src/Features/Core/Portable/Navigation/NavigationOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Navigation
+{
+    internal static class NavigationOptions
+    {
+        public const string FeatureName = "Navigation";
+
+        public static readonly Option<bool> UsePreviewTab = new Option<bool>(FeatureName, "UsePreviewTab", defaultValue: false);
```

Comment:
No, this is an option that gets passed around when calling IDocumentNavigationService to use the provisional tab. Previously, there was a "usePreviewTab" parameter. Since I'm adding an OptionSet parameter to make NavigateToObjectBrowser work, it seemed good to take care of this old flag at the same time.